### DISCUSSION TITLE
Considering make a Vue version?

### DISCUSSION THREAD

**Discussion by orangeagain** [jeffreylanters/react-unity-webgl#387] [Feature Requests]
The [[vue-unity-webgl](https://github.com/votetake/vue-unity-webgl)](https://github.com/votetake/vue-unity-webgl/issues/27) is dead for Unity 2020.
many people want a vue version, even willing pay for that. anyone want to move this library to vue?

**Comment by jeffreylanters** [comment]
I did notice the lack of versions for other libraries, and have considered making Vue and Angular versions respectively about a year ago.

If there is enough interest, I'll look into it.
